docs/linkify-angular-ts.md

```diff
@@ -0,0 +1,251 @@
+---
+layout: doc
+title: [ngx-linkifyjs](https://github.com/AnthonyNahas/ngx-linkifyjs) · Documentation
+---
+
+`ngx-linkifyjs` is an angular V7 wrapper for linkifyjs -  library for finding links in plain text and converting them 
+to HTML <a> tags via linkifyjs and more <3
+
+#### Jump to
+
+* [Demo](#demo)
+* [Installation](#installation)
+* [Usage](#usage)
+  * [Pipe](#pipe) `NgxLinkifyjsPipe via | linkify`
+  * [Service](#service) `NgxLinkifyjsService`
+    * [Methods](#methods)
+        * [linkify _(text: string): string_](#linkify_method)
+        * [find](#find_method)
+        * [test](#test_method)
+  * [Configuration](#configuration)
+* [Documentation](#documentation)
+
+
+## Demo
+
+View all the directives in action at [https://anthonynahas.github.io/ngx-linkifyjs](https://anthonynahas.github.io/ngx-linkifyjs)
+
+## Installation
+
+
+Install above dependencies via *npm*. 
+
+Now install `ngx-linkifyjs` via:
+```shell
+npm i -s ngx-linkifyjs
+```
+
+---
+##### SystemJS
+>**Note**:If you are using `SystemJS`, you should adjust your configuration to point to the UMD bundle.
+In your systemjs config file, `map` needs to tell the System loader where to look for `ngx-linkifyjs`:
+```js
+map: {
+  'ngx-linkifyjs': 'node_modules/ngx-linkifyjs/bundles/ngx-linkifyjs.umd.js',
+}
+```
+---
+
+Once installed you need to import the main module:
+```js
+import { NgxLinkifyjsModule } from 'ngx-linkifyjs';
+```
+The only remaining part is to list the imported module in your application module. The exact method will be slightly
+different for the root (top-level) module for which you should end up with the code similar to (notice ` NgxLinkifyjsModule .forRoot()`):
+```typescript
+import { NgxLinkifyjsModule } from 'ngx-linkifyjs';
+
+@NgModule({
+  declarations: [AppComponent, ...],
+  imports: [NgxLinkifyjsModule.forRoot(), ...],  
+  bootstrap: [AppComponent]
+})
+export class AppModule {
+}
+```
+
+Other modules in your application can simply import ` NgxLinkifyjsModule `:
+
+```js
+import { NgxLinkifyjsModule } from 'ngx-linkifyjs';
+
+@NgModule({
+  declarations: [OtherComponent, ...],
+  imports: [NgxLinkifyjsModule, ...], 
+})
+export class OtherModule {
+}
+```
+
+
+## Usage
+
+
+Once the library is imported, you can use its components, directives and pipes in your Angular application:
+
+### Pipe
+
+`{{text | linkify}}`
+
+```html
+<span [innerHTML]="'Linkify the following URL: https://github.com/anthonynahas/ngx-linkifyjs and share it <3' | linkify"></span>
+```
+
+**result**: Linkify the following URL: [https://github.com/anthonynahas/ngx-linkifyjs](https://github.com/anthonynahas/ngx-linkifyjs) and share it <3
+
+### Service
+
+Inject the `NgxLinkifyjsService` service
+
+```typescript
+import {NgxLinkifyjsService, Link, LinkType} from 'ngx-linkifyjs';
+
+constructor(public linkifyService: NgxLinkifyjsService) {
+ } 
+}
```

Comment:
Extra curly here. 

Is the indentation correct? In Angular, can you have raw `constructor`s like this?